os/kernel/binary_manager/binary_manager.c

```diff
@@ -74,18 +80,25 @@ mqd_t binary_manager_get_mqfd(void)
  ****************************************************************************/
 int binary_manager(int argc, char *argv[])
 {
+#ifdef CONFIG_APP_BINARY_SEPARATION
 	int ret;
```

Comment:
If we move this declaration under `request_msg`, we can merge the ifdef with `binary_manager_scan_ubin_all`.
It looks much simple.